src/librustc_mir/borrow_check/nll/universal_regions.rs

```diff
@@ -64,12 +69,14 @@ pub struct UniversalRegions<'tcx> {
     /// closure type, but for a top-level function it's the `TyFnDef`.
     pub defining_ty: Ty<'tcx>,
 
-    /// The return type of this function, with all regions replaced
-    /// by their universal `RegionVid` equivalents.
+    /// The return type of this function, with all regions replaced by
```

Comment:
I think you mean "This type contains explicit free regions" ?

In my mind;
1. Normalization of types refers to associated types.
2. Replacing explicit free regions with region numbers is basically the *opposite* of normalization - any explicit region can be replaced with multiple region numbers.
at least in my mind, "normalization" of types mostly refers to associated types, and replacing explicit ).